src/pages/Home/Home.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import Cards from '../../components/Card/Cards';
+import { allCoins } from '../../redux/home-reducer';
+import logo from '../../images/1960x0.jpg';
+
+const Home = () => {
+  const [search, setSearch] = useState('');
+  const handleSearch = (e) => {
+    setSearch(e.target.value);
+  };
+  const Coins = useSelector((state) => state.coin);
```

Comment:
- [ ] Good job trying to make some of the changes required by the previous reviewer, but I noticed that there is still some thing you should improve such the background colors where one card has a background with `light` color and the other one in `dark` color.

| Template design | Your Design |
| --------------------- | ---------------------- |
| ![Screenshot from 2023-02-10 22-09-20](https://user-images.githubusercontent.com/83042742/218188764-ae8f8979-97ec-497e-bd57-beb0a4119a7e.png) | ![Screenshot from 2023-02-10 22-04-13](https://user-images.githubusercontent.com/83042742/218188532-a2d04753-7bed-4c32-a5c3-e8c5cbbb1a83.png) |